regression-tests.dnsdist/runtests

```diff
@@ -49,12 +49,19 @@ openssl x509 -req -days 1 -CA ca.pem -CAkey ca.key -CAcreateserial -in server.cs
 # Generate a chain
 cat server.pem ca.pem > server.chain
 
-if ! nosetests --with-xunit $@; then
+out=$(mktemp)
+set -o pipefail
+if ! nosetests --with-xunit $@ 2>&1 | tee "${out}" ; then
     for log in configs/*.log; do
         echo "=== ${log} ==="
         cat "${log}"
+	echo
```

Comment:
Not sure about that one (the indentation is off, at least).

Reply:
For me the blank line really makes it easier to group the lines belonging to a specific test. I fixed the indentation.

Reply:
Fine by me!